### ISSUE TITLE
Bug - Transients created in autoload=true, on table wp_options

### ISSUE THREAD

**Issue by algmelo** [Automattic/fb-instant-articles#901]
## Steps required to reproduce the problem

1. Insert a embed on post content (Youtube, Facebook, Twitter or Instagram)
2. Access this post
3. Verify MySQL table wp_options, and run this query:

SELECT option_name, option_value
FROM wp_options
WHERE autoload = 'yes'

4. See the result, and you'll find something like this:

option_name                                                                     option_value
_transient_c80eac53c8183dd70511050453ff1b33      youtube
_transient_c80eac53c89340d9dfd49d8gddf42d94   twitter

## Expected Result

The expected result is that this item is without autoload = true, which is overloading the mysql response, which needs to return a huge amount of completely unnecessary information.

## Version Info

Please state _exact_ versions, not _latest_ or _new_.

* Plugin version: 4.0.6
* WordPress version: 4.9.4
* PHP version: 5.5.9

Without the above information we are not able to debug the issue and will close it.

**Comment by algmelo** [comment]
No news on this bug?

**Comment by everton-rosario** [comment]
@algmelo Not sure I fully understood the problem you are raising here.
Are you affirming that the `autoload = true` is being set by this plugin?

**Comment by everton-rosario** [comment]
Gotcha.
I could find the lines you should try change:

File `embed.php`
```
	if ( $provider_name ) {
		$html = instant_articles_embed_get_html( $provider_name, $html, $url, $attr, $post_id );
		delete_transient( $cache_key );   // Add this line right after the get_html
	}
```

Didn't test, but it looks promising.
Check if this will fix the issue for you.

**Comment by algmelo** [comment]
Great, @everton-rosario !!!

This solved the problem, but how to do in case of future updates in the plugin?

Thanks!

**Comment by everton-rosario** [comment]
Can you send a PR with that fix? We need to double check for further impact into the code (if this will generate any unwanted collateral behavior).

**Comment by algmelo** [comment]
Hi, @everton-rosario 

I created a PR with the fix. I had never done a PR haha

I hope the amendment is accepted.

Thanks for the guidance!

**Comment by GaryJones** [comment]
The PR was merged and released in 4.1.1.
